test/jdk/javax/script/MyContext.java

```diff
@@ -49,7 +49,7 @@ public MyContext() {
         appScope = new SimpleBindings();
         engineScope = new SimpleBindings();
         globalScope = null;
-        reader = new InputStreamReader(System.in);
+        reader = new InputStreamReader(System.in, System.getProperty("stdin.encoding"));
```

Comment:
I will revert this change, since

1. AFAICT, this file is not used anymore
2. Neither `std{out,err}.encoding` were used while creating `PrintWriter`s below

Reply:
Reverted in ca96e12be8e.